src/lib/parse/forbiddenChars.cpp

```diff
@@ -61,3 +61,36 @@ bool forbiddenChars(const char* s)
 
   return false;
 }
+
+
+
+/* ****************************************************************************
+*
+* forbiddenCharsInUriParam - 
+*/
+bool forbiddenCharsInUriParam(const char* s)
```

Comment:
The name forbiddenCharsInUriParam() is too generic, although the "straighforward" alternative is ugly: forbiddenCharsInGeomAndCoords().

Maybe forbiddenCharsInUriParamWithoutSemicolon() is better?